pkg/sync/v9/translate.go

```diff
@@ -326,7 +326,7 @@ var translations = map[string][]struct {
 		{
 			Path: jsonpath.MustCompile("$.metadata.labels['azure.openshift.io/sync-pod-optionally-apply']"),
 			F: func(cs *api.OpenShiftManagedCluster, o interface{}) (interface{}, error) {
-				if cs.Properties.MonitorProfile.WorkspaceID != "" && cs.Properties.MonitorProfile.WorkspaceKey != "" {
```

Comment:
do we have some validation that says "if enabled and other workspace fields are not set, error"?

Reply:
We don't and it sounds like a good idea. I'll add it. :+1: